src/simulator/distance_measuring_sensor.py

```diff
@@ -13,26 +13,29 @@ class DistanceMeasuringSensor(sensor.Sensor):
     it is a sensor that measures the distance of an object to where the sensor is located.
     """
     def __int__(self, oid: object.ObjectId, name: str, pos: position.Position,
-                owner_object: object.Object, registry: object_registry.ObjectRegistry, parameter: float):
+                owner_object: object.Object, registry: object_registry.ObjectRegistry, sensor_range: float):
         self.position = pos
```

Comment:
make a copy. It's dangerous this way. If someone changes pos in the calling function, then this will also change.